app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -3,7 +3,7 @@
     Mod Center
   </h1>
   <nav class="mod-center--nav" aria-label="Mod center inbox navigation">
-    <h3 class="fw-bold fs-s mod-center--nav__title">
+    <h3 class="crayons-link--header">
```

Comment:
Since it's not a link, should it be called "crayons-link"?

I imagine it could be used in different scenarios in sidebar not only in context of links (think of form filters etc..)

Maybe simply `crayons-header`? Or actually `crayons-subheader`  since we may turn all page titles into its own component at some point too?

Reply:
Good catch!

If it is `crayons-subheader`, which file do you think would be an appropriate one of put this? 
And re: `crayons-header`, I think it could be handy to define this as well, since we want to have the same consistent type styling for all page titles :)

Reply:
I think you can create new component file: `components/headers.scss` and put your code for `.crayons-subheader`. I can then (in a separate PR) add styling for `crayons-header` too :)